cmd/influxd/main.go

```diff
@@ -78,11 +74,13 @@ func main() {
 	case "":
 		execRun(args)
 	case "backup":
+		setGoMaxProcs()
```

Comment:
I don't think it's necessary. Backup & restore is single threaded since it builds a `tar` archive sequentially.